test/e2e/storage/vsphere/vsphere_volume_vsan_policy.go

```diff
@@ -111,7 +107,7 @@ var _ = utils.SIGDescribe("Storage Policy Based Volume Provisioning [Feature:vsp
 			e2elog.Failf("Unable to find ready and schedulable Node")
 		}
 		masternodes, _ := framework.GetMasterAndWorkerNodesOrDie(client)
-		gomega.Expect(masternodes).NotTo(gomega.BeEmpty())
+		gomega.Expect(masternodes).NotTo(gomega.BeEmpty(), "cluster has no master nodes")
```

Comment:
nit: repo style generally is to dot import gomega and ginkgo to avoid verbosity for `Expect` and `BeEmpty` etc.